pkgs/by-name/ju/jujutsu/package.nix

```diff
@@ -0,0 +1,124 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  installShellFiles,
+  pkg-config,
+  zstd,
+  libgit2,
+  libssh2,
+  openssl,
+  darwin,
+  libiconv,
+  git,
+  gnupg,
+  openssh,
+  buildPackages,
+  nix-update-script,
+  testers,
+  jujutsu,
+}:
+
+let
+  version = "0.21.0";
+in
+
+rustPlatform.buildRustPackage {
+  pname = "jujutsu";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "martinvonz";
+    repo = "jj";
+    rev = "v${version}";
+    hash = "sha256-uZsfHhcYpobatWaDQczuc9Z3BWHN5VO0qr/8mu5kEio=";
+  };
+
+  cargoHash = "sha256-BOO1jP1Y5CNbE97zj+tpariiBdcuxKb1wyvI7i/VpYI=";
+
+  nativeBuildInputs = [
+    installShellFiles
+    pkg-config
+  ];
+
+  buildInputs =
+    [
+      zstd
+      libgit2
+      libssh2
+    ]
+    ++ lib.optional (!stdenv.hostPlatform.isDarwin) [ openssl ]
```

Comment:
```suggestion
    ++ lib.optional (!stdenv.hostPlatform.isDarwin) openssl
```

```suggestion
    ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [ openssl ]
```

Reply:
Thanks, typo 🫠 Fixed!

We need to lint for this…